sqldelight-gradle-plugin/src/test/fixtures/well-formed-selects/expected/com/sample/HockeyPlayerModel.java

```diff
@@ -622,6 +512,14 @@ public Marshal marshal(HockeyPlayerModel copy) {
       return new Marshal(copy, birth_dateAdapter, shootsAdapter, positionAdapter);
     }
 
+    public SqlDelightStatement select_all() {
+      return new SqlDelightStatement(""
+          + "SELECT *\n"
+          + "FROM hockey_player\n"
+          + "JOIN team ON hockey_player.team = team._id",
+      new String[0], Collections.<String>unmodifiableSet(new LinkedHashSet<String>(Arrays.asList("hockey_player","team"))));
```

Comment:
Ugh, this is depressing. In 10 years we can use the new factories going into Java 9.